app/helpers/turnarounds.js

```diff
@@ -136,14 +136,15 @@ const timelines = pastLaunches => {
     options
   };
 
-  const lastLaunchDate = pastLaunches[pastLaunches.length - 1].launch_date_utc;
```

Comment:
Did the past laucnhes order has changed? It seems that it is always pointing to the first Falcon 1 launch